apollo-backend/src/main/java/io/logz/apollo/controllers/ScalingController.java

```diff
@@ -82,6 +90,24 @@ public void updateScalingFactor(int groupId, int scalingFactor, Req req) {
         group.setScalingStatus(Group.ScalingStatus.PENDING);
         groupDao.updateGroup(group);
 
+        String userEmail = req.token(Tokens._USER).toString();
+        Service service = serviceDao.getService(group.getServiceId());
+        Environment environment = environmentDao.getEnvironment(group.getEnvironmentId());
+
+        MDCLogging.withMDCFields(ImmutableMap.of("userEmail", userEmail,
+                                                "groupName", group.getName(),
+                                                "serviceName", service.getName(),
+                                                "envName", environment.getName(),
```

Comment:
environmentIName :)

Reply:
Try writing that huge field when you query, see if you avoid misspelling it ;)